inlong-sort/sort-connectors/oracle-cdc/pom.xml

```diff
@@ -44,6 +44,11 @@
             <artifactId>sort-connector-base</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.inlong</groupId>
```

Comment:
The dependency is exists in the pom.xml of connector-base.

Reply:
I suggest that write this module directly. Because this module use audit sdk